hack/build-docker.sh

```diff
@@ -22,48 +22,13 @@ set -e
 source hack/common.sh
 source hack/config.sh
 
-if [ -z "$1" ]; then
-    target="build"
-else
-    target=$1
-    shift
-fi
-
 if [ $# -eq 0 ]; then
     args=$docker_images
```

Comment:
I guess we can remove the docker_images variable from hack/config-defaults.sh. Should be unused at that point.

Reply:
Oh  is it still used the pre-pull the images into the test nodes? Maybe we can have a more alligned way with bazel.